addons/account/views/res_company_views.xml

```diff
@@ -8,6 +8,16 @@
             <xpath expr="//field[@name='country_id']" position="after">
                 <field name="country_code" invisible="1"/>
             </xpath>
+            <xpath expr="//notebook" position="inside">
+                <page string="Accounting Information">
+                    <group>
+                        <field name="income_currency_exchange_account_id"
+                                               groups="account.group_account_readonly"/>
+                        <field name="expense_currency_exchange_account_id"
+                                               groups="account.group_account_readonly"/>
+                    </group>
+                </page>
+            </xpath>
```

Comment:
Don't add this into the company. Instead, add related fields to settings wizard and then, add these fields above the configuration of the Exchange Difference Journal there:
https://github.com/odoo/odoo/blob/master/addons/account/views/res_config_settings_views.xml#L167